cgi-bin/LJ/Faq.pm

```diff
@@ -88,7 +91,7 @@ sub load {
         return undef unless $f;
         $faq = $class->new(%$f, lang => $lang);
 
-    } else { # Don't load fields that lang_update_in_place will overwrite.
+    } elsif ($rev == 0) { # Don't load fields that lang_update_in_place will overwrite.
```

Comment:
Okay, so there are two cases in this if statement already: loading a faq in the site's default language, and loading a faq in some other language. Adding this elsif here gives you three cases: loading a faq in the site's default language, loading a faq in some other language at revision 0, and loading a faq in some other language at a nonzero revision. Shouldn't there be a fourth case for loading a faq in the site's default language at a nonzero revision?